modules/cij/runner.py

```diff
@@ -572,7 +630,7 @@ def tsuite_enter(trun: TestRun, tsuite: TestSuite) -> int:
     return rcode
 
 
-def tsuite_setup(trun: TestRun, declr, enum) -> TestSuite:
+def tsuite_setup(trun: TestRun, tplan, declr, enum) -> TestSuite:
```

Comment:
Since we're already type hinting most of our own classes, I think we should add it to `tplan: TestPlan` here as well

Reply:
fixed.